docs/assets/css/secondary-nav.less

```diff
@@ -1,85 +1,178 @@
+            });
+        }
 
-    &_item {
-        list-style: none;
+        .o-expandable_content__collapsed {
+            .respond-to-min(901px, {
+                max-height: initial !important;
+                overflow: visible;
+            });
+        }
+    }
 
-        .respond-to-min(801px, {
-            & + & {
-                margin-top: 10px;
+    &.o-expandable__padded {
+        .o-expandable_content {
+            &:before {
+                .respond-to-min(901px, {
+                    border: none;
+                });
             }
-        });
+        }
     }
 
-    &_link {
-        @font-size-small: 16px;
-        @font-size-large: 18px;
-
-        .respond-to-max(800px, {
-            display: block;
-            // width: 100%; // Needed to override button styles
-            padding: unit((@grid_gutter-width / 2) / @font-size-small, em)
-                     unit(@grid_gutter-width / @font-size-small, em);
-            border-bottom-style: solid;
-            border-width: 0; // Needed to override button styles
-            border-bottom-width: 1px;
-            background: @gray-10;
-            font-size: unit(@font-size-small / @base-font-size-px, em);
-            font-weight: 500;
-            .u-link__colors(@dark-gray, @dark-gray, @dark-gray, @dark-gray, @dark-gray,
-                            @gray-20, @gray-20, @gray-20, @gray-20, @gray-20);
-            text-align: left; // Needed to override button styles
+    &.o-expandable__background {
+        .respond-to-min(901px, {
+            background: transparent;
         });
+    }
 
-        .respond-to-max(600px, {
-            padding-left: unit((@grid_gutter-width / 2) / @font-size-small, em);
-            padding-right: unit((@grid_gutter-width / 2) / @font-size-small, em);
-        });
+}
+
+.nav-secondary {
+    ul > li > .h5 {
+        color: #676A6F;
+        font-weight: 500;
+    }
+    .m-list__links {
 
-        .respond-to-min(801px, {
-            display: inline-block;
-            padding: unit(5px / @font-size-large, em)
-                     unit(15px / @font-size-large, em);
-            border-left-style: solid;
-            border-left-width: 5px;
-            font-size: unit(@font-size-large / @base-font-size-px, em);
-            font-weight: 500;
-            .u-link__colors(@pacific, @pacific, @black, @black, @black,
-                            transparent, transparent, @green, @green, @green);
+        .respond-to-min(901px, {
+                margin-top: unit( 5px / @base-font-size-px, em );
         });
 
-        &__current {
-            .respond-to-max(800px, {
-                display: none;
+        .m-list_item {
+            .respond-to-min(901px, {
+                margin-bottom: unit( 4px / @base-font-size-px, em );
+            });
+        }
+
+        .m-list_link {
+            padding-left: unit( 20px / @base-font-size-px, em );
+            .respond-to-max(900px, {
```

Comment:
Should this use the capital framework breakpoint less variable https://github.com/cfpb/capital-framework/blob/master/packages/cf-core/src/cf-vars-breakpoints.less#L11 ?